front/app/containers/InitiativesShow/ReactionControl/Status/ProposedNotReacted.tsx

```diff
@@ -244,17 +236,12 @@ const ProposedNotReacted = ({
             disabledReason ? disabledReason : ''
           }`}
         >
-          <StyledButton
-            icon="vote-up"
-            aria-describedby="tooltip-content"
-            disabled={!!tippyContent}
-            buttonStyle="primary"
-            onClick={onReaction}
-            id="e2e-initiative-like-button"
-            ariaDisabled={false}
-          >
-            <FormattedMessage {...messages.vote} />
-          </StyledButton>
+          <VoteButtons
+            voteButtonId="e2e-initiative-like-button"
+            onCancelReaction={onCancelReaction}
+            onReaction={onReaction}
+            userReacted={userReacted}
+          />
```

Comment:
Because userReacted is always false when this component is rendered, it only shows the Vote button, not the Voted button. Also note this component will be merged with ProposedReacted into one component in a later PR.